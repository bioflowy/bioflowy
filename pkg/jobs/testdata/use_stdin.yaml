children:
  - args:
      - "sh"
      - "-c"
      - "echo  'this is stdin test' > $(pipe)"
    outputs:
      - "pipe"
  - args:
      - "sh"
      - "-c"
      - "nl > use_stdin.yaml.result"
    stdin:
      pipe: "pipe"
