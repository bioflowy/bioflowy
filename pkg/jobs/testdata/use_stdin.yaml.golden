     1	this is stdin test
